src/schema/__tests__/partner_shows.test.js

```diff
@@ -7,49 +7,22 @@ describe("PartnerShows type", () => {
 
   beforeEach(() => {
     const gravity = sinon.stub()
-
```

Comment:
Why is this loader not being replaced?

Reply:
Yea, there's still some tricky ones left. This one actually has a URL that's derived from _2_ id's, its a very nested route.

Like `/partner/:partner_id/show/:show_id/artworks`, etc.

Reply:
Ah, but you can pass multiple IDs into the loader callback… I think?

```js
showArtworksLoader(({ partner_id, show_id }) => `/partner/${partner_id}/show/${show_id}/artworks`)
```